installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,215 @@
+include::modules/oc-mirror-about.adoc[leveloffset=+1]
+
+// About V2
+include::modules/oc-mirror-v2-about.adoc[leveloffset=+2]
+
+// oc-mirror compatibility and support
+include::modules/oc-mirror-support.adoc[leveloffset=+1]
+
+// About the mirror registry
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For information about viewing the CRI-O logs to view the image source, see xref:../../installing/validating-an-installation.adoc#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+
+[id="prerequisites_installing-mirroring-disconnected"]
+== Prerequisites
+
+* You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2[Docker v2-2] in the location that will host the {product-title} cluster, such as Red Hat Quay.
++
+[NOTE]
+====
+If you use Red Hat Quay, you must use version 3.6 or later with the oc-mirror plugin. If you have an entitlement to Red Hat Quay, see the documentation on deploying Red Hat Quay link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/[for proof-of-concept purposes] or link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/deploying_the_red_hat_quay_operator_on_openshift_container_platform/index[by using the Red Hat Quay Operator]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat Support.
+====
++
+If you do not already have an existing solution for a container image registry, subscribers of {product-title} are provided a link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/installing/disconnected-installation-mirroring#installing-mirroring-creating-registry[mirror registry for Red Hat OpenShift]. The _mirror registry for Red Hat OpenShift_ is included with your subscription and is a small-scale container registry that can be used to mirror the required container images of {product-title} in disconnected installations.
+
+* Save the file as `$XDG_RUNTIME_DIR/containers/auth.json`.
+
+
+[id="using-oc-mirror"]
+== Using oc-mirror
+
+The instructions in this section will show how to prepare the environment and use oc-mirror.
+
+.Environment Preparation
+
+* To set up the credentials for pulling/pushing container images, download the pull secret and place it in the correct path.
+
+** The default podman credentials location `$XDG_RUNTIME_DIR/containers/auth` is used for authenticating to the registries.
+** The docker location `~/.docker/config.json` for credentials is also supported.
+
+[Optional Step] 
+* Use the following command to run a local registry that will be used as a destination:
+
+[source,terminal]
+----
+$ podman run -dt -p 6000:5000 -e REGISTRY_STORAGE_DELETE_ENABLED=true --name local-registry docker.io/library/registry:2
```

Comment:
not sure we need this optional step, especially that there is a lot of content in prerequisite, and aboute the mirror registry to explain how to setup a highly available registry.
we don' t want to tell users to have a local (not hightly available) registry for their production mirroring.

Reply:
+1, i agree

Reply:
Ack